README.md

```diff
@@ -8,45 +8,271 @@
+
+### Step 4: Setup Authentication
+
+1. Move to the `Auth` tab.
+   ![Configure authentication](./docs/resources/config-auth.png)
+
+2. In the `Scopes` section, add the following scopes for your app using the `Add new scopes` button.
+   - `e-commerce`
+   - `tickets`
+   - `crm.objects.goals.read`
+   - `media_bridge.read`
+   - `crm.objects.custom.read`
+   - `crm.objects.custom.write`
+   - `crm.objects.feedback_submissions.read`
+
+   ![Add scopes](./docs/resources/add-scopes.png)
+
+3. In the `Redirect URL` section, add the redirect URL for your app. This is the URL where the user will be redirected after the authentication process. You can use `localhost` for testing purposes. Then click the `Create App` button.
+
+   ![Redirect URL](./docs/resources/redirect-url.png)
+
+### Step 5: Get the Client ID and Client Secret
+
+Navigate to the `Auth` tab and you will see the `Client ID` and `Client Secret` for your app. Make sure to save these values.
+![Client ID and Client Secret](./docs/resources/client-id-secret.png)
+
+### Step 6: Setup Authentication Flow
+
+Before proceeding with the Quickstart, ensure you have obtained the Access Token using the following steps:
+
+1. Create an authorization URL using the following format:
+
+   ```
+   https://app.hubspot.com/oauth/authorize?client_id=<YOUR_CLIENT_ID>&scope=<YOUR_SCOPES>&redirect_uri=<YOUR_REDIRECT_URI>
+   ```
+
+   Replace the `<YOUR_CLIENT_ID>`, `<YOUR_REDIRECT_URI>` and `<YOUR_SCOPES>` with your specific value.
+2. Paste it in the browser and select your developer test account to intall the app when prompted.
+   !![Account select](./docs/resources/account-select.png)
+
+3. A code will be displayed in the browser. Copy the code.
+
+4. Run the following curl command. Replace the `<YOUR_CLIENT_ID>`, `<YOUR_REDIRECT_URI`> and `<YOUR_CLIENT_SECRET>` with your specific value. Use the code you received in the above step 3 as the `<CODE>`.
+
+   - Linux/macOS
+
+     ```bash
+     curl --request POST \
+     --url https://api.hubapi.com/oauth/v1/token \
+     --header 'content-type: application/x-www-form-urlencoded' \
+     --data 'grant_type=authorization_code&code=<CODE>&redirect_uri=<YOUR_REDIRECT_URI>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+     ```
+   - Windows
+
+     ```bash
+     curl --request POST ^
+     --url https://api.hubapi.com/oauth/v1/token ^
+     --header 'content-type: application/x-www-form-urlencoded' ^
+     --data 'grant_type=authorization_code&code=<CODE>&redirect_uri=<YOUR_REDIRECT_URI>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+     ```
+
+   This command will return the access token necessary for API calls.
+
+   ```json
+   {
+     "token_type": "bearer",
+     "refresh_token": "<Refresh Token>",
+     "access_token": "<Access Token>",
+     "expires_in": 1800
+   }
+   ```
+5. Store the access token securely for use in your application.
 
 ## Quickstart
```

Comment:
Could you please restructure the `Quickstart` guide by following the same structure used in the following repo:

https://github.com/ballerina-platform/module-ballerinax-hubspot.marketing.emails/blob/a8f4e2ddbd49f1addbbb264f12a6355107437857/ballerina/Package.md#quickstart